src/explore-education-statistics-admin/src/pages/documentation/DocumentationStyle.tsx

```diff
@@ -1050,6 +1071,15 @@ const DocumentationGlossary = () => {
               <p>Lower case.</p>
               <h3 id="national-scholarship-fund">national scholarship fund</h3>
               <p>Lower case.</p>
+              <h3 id="national-statistics">National Statistics</h3>
+              <p>
+                Upper case National Statistics for the official statistics
+                quality mark.{' '}
```

Comment:
a lot of paragraphs are ending with an unnecessary space `{' '}` please _Find_ them all and determine if they need to be there, remove if not